<?xml version="1.0"?>
<launch>
    <arg name="bag_dir" default="$(find insta360_ros_driver)/bag"/>
    <arg name="bag_type" default="compressed"/>
    <arg name="bag_file" default="record.bag"/>
    
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find insta360_ros_driver)/config/preview.rviz" output="screen"/>

    <node name="rosbag" pkg="rosbag" type="play" args="--clock $(arg bag_dir)/$(arg type)/$(arg bag_file)" output="screen"/>
</launch>